docs/content/guides/dagster-pipes.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Dagster Pipes | Dagster Docs
+description: "Dagster Pipes provides a protocol between the orchestration environment (Dagster) and external execution (ex: Databricks) and a toolkit for building implementations of that protocol."
+---
+
+# Dagster Pipes
+
+At a high level, Dagster Pipes allows you to send Dagster context data into an external process which is then sent back to Dagster.
```

Comment:
Btw I had written this is Notion as the first few paragraphs:

"Dagster Pipes is a toolkit for building integrations between Dagster and external execution environments. It standardizes the process of passing parameters, injecting context information, ingesting logs, and collecting metadata all while remaining agnostic to how those remote computations are launched in those environments. This enables the clean separation of orchestration and business logic in the Dagster ecosystem.

It also smooths the process of incorporating pre-existing code, business logic, and execution environments into Dagster. Using Dagster Pipes, you add just a few lines to you code, and your code becomes seamlessly invocable by the orchestrator, and in turn can stream logs and metadata back to Dagster so you can leverage its observability, lineage, cataloging, and debugging capabilities."

Reply:
thanks! I started moving this right before you added that. I updated this.